.github/workflows/release.yml

```diff
@@ -14,27 +14,27 @@ jobs:
       uses: actions/checkout@v2
       with:
         submodules: recursive
-    - run: git clone --depth 1 --branch solana-rustc/14.0-2022-03-22 https://github.com/solana-labs/llvm-project.git
+    - run: git clone --depth 1 --branch solana-rustc/15.0-2022-08-09 https://github.com/solana-labs/llvm-project.git
```

Comment:
If we're using solang-llvm when the release workflow should not build llvm any more